src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -284,10 +284,10 @@
             <% }) %>
             <% if (legend.length > 5) { %>
                 <div class="item expand">
-                    <a href="#" class="expand-legend">View <% print(legend.length - 5) %> more</a>
+                    <a href="#" class="expand-legend"><span class="i18n" data-i18n="View">View</span> <% print(legend.length - 5) %> <span class="i18n" data-i18n="more">more</span></a>
```

Comment:
Theoretically we could also consider rendering this entire thing via some kind of helper function, but I don't know that that's idiomatic to C# templates.